tests/bot/exts/moderation/test_incidents.py

```diff
@@ -768,3 +787,73 @@ async def test_on_message_non_incident(self):
             await self.cog_instance.on_message(MockMessage())
 
         mock_add_signals.assert_not_called()
+
+
+class TestMessageLinkEmbeds(TestIncidents):
+    """Tests for `extract_message_links` coroutine."""
+
+    async def test_shorten_text(self):
+        """Test all cases of text shortening by mocking messages."""
+        tests = {
+            "thisisasingleword"*10: ('thisisasingleword'*10)[:50]+"...",
+            "\n".join("Lets make a new line test".split()): "Lets\nmake\na"+"...",
+            'Hello, World!' * 300: ('Hello, World!' * 300)[:300] + '...'
+        }
+
+        for test, value in tests.items():
+            self.assertEqual(
+                str(incidents.shorten_text(test)), value
+            )
```

Comment:
You should use subtests here. You'll find an example in the test module's README.

Reply:
Resolved in 8e39adae.